decidim-sortitions/spec/services/decidim/sortitions/admin/draw_spec.rb

```diff
@@ -27,7 +27,9 @@ module Admin
         end
 
         it "Draw can be reproduced several times" do
+          # rubocop:disable RSpec/IdenticalEqualityAssertion
```

Comment:
Instead of disabling the rule here, could we store the initial value as `let(:initial_draw) { Draw.for(sortition) }` and then change the expectation to `eq(initial_draw)`?

Reply:
It makes sense